### PR TITLE
Update speakers page design

### PR DESCRIPTION
**Pull request by MarioRodrigues10**

### CODE REVIEW

File: lib/atomic_web/live/speaker_live/show.ex [hunk 1]
```diff
@@ -10,9 +10,29 @@ defmodule AtomicWeb.SpeakerLive.Show do
 
   @impl true
   def handle_params(%{"id" => id}, _, socket) do
```
Comment (feliciofilipe) [inline lib/atomic_web/live/speaker_live/show.ex:12]:
```suggestion
  def handle_params(%{"id" => id}, _, socket) do
  
      speaker = Activities.get_speaker!(id)
```

File: lib/atomic_web/live/speaker_live/show.ex [hunk 2]
```diff
@@ -10,9 +10,29 @@ defmodule AtomicWeb.SpeakerLive.Show do
...
+    entries = [
+      %{
+        name: gettext("Speakers"),
+        route: Routes.speaker_index_path(socket, :index)
+      },
+      %{
+        name: "Show Speaker",
```
Comment (feliciofilipe) [inline lib/atomic_web/live/speaker_live/show.ex:19]:
```suggestion
        name: speaker.name,
```

File: lib/atomic_web/live/speaker_live/show.ex [hunk 3]
```diff
@@ -10,9 +10,29 @@ defmodule AtomicWeb.SpeakerLive.Show do
...
+      activities_sessions
+      |> Enum.map(fn activity -> Activities.get_activity!(activity.activity_id) end)
+
     {:noreply,
      socket
+     |> assign(:current_page, :speakers)
      |> assign(:page_title, page_title(socket.assigns.live_action))
+     # |> assign(:activities, activities)
+     |> assign(:breadcrumb_entries, entries)
      |> assign(:speaker, Activities.get_speaker!(id))}
```
Comment (feliciofilipe) [inline lib/atomic_web/live/speaker_live/show.ex:36]:
```suggestion
     |> assign(:speaker,  speaker)}
```

**Comment by MarioRodrigues10** [comment]
Test this version, later i'll update to mobile because there's a mobile issue at the moment with navbar.

File: priv/repo/seeds/activities.exs
Comment (MarioRodrigues10) [inline priv/repo/seeds/activities.exs:1]: I'm sorry, but the information in this file appears to be unrelated or disconnected from the current context.

**Comment by MarioRodrigues10** [comment]
We are currently not using this branch anymore.
This page is already done in develop branch
